src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -137,3 +143,199 @@ public sealed class SerializationInfoEnumerator : IEnumerator
         public void Reset() { throw null; }
     }
 }
+namespace System.Runtime.Serialization
+{
+    [System.CLSCompliantAttribute(false)]
```

Comment:
CLSCompliant? No.  If something isn't compliant and isn't attributed to say so, it'll produce a build error.